src/shared-components/accordion/test.tsx

```diff
@@ -7,26 +7,22 @@ import Accordion from './index';
 const testAccordion = {
   title: <div>title</div>,
   isOpen: false,
-  onClick: () => {},
+  // eslint-disable-next-line @typescript-eslint/no-empty-function
```

Comment:
We can remove this by defining `onClick` like so: `onClick: (): void => undefined,`

Reply:
done